src/main/java/com/cleanroommc/multiblocked/core/asm/DynamicTileEntityGenerator.java

```diff
@@ -40,7 +40,7 @@ public DynamicTileEntityGenerator(String name, List<CapabilityTrait> traits, Cla
     }
 
     public Class<?> generateClass() {
-        ClassWriter classWriter = new ClassWriter(0);
+        ClassWriter classWriter = new ClassWriter(ClassWriter.COMPUTE_FRAMES | ClassWriter.COMPUTE_MAXS);
```

Comment:
I don't know if there's a downside to this, but it means you don't need to worry about setting the correct values via `visitVarMaxs()`. I couldn't find the correct combination there that worked.